pom.xml

```diff
@@ -67,8 +67,8 @@
         <jenkins.version>2.346.1</jenkins.version>
         <no-test-jar>false</no-test-jar>
         <groovy-cps.version>1.32</groovy-cps.version>
-        <node.version>12.19.0</node.version>
-        <npm.version>6.14.8</npm.version>
+        <node.version>16.17.0</node.version>
+        <npm.version>8.18.0</npm.version>
```

Comment:
Aligns with the node version I'm using in core, which is the current LTS line. npm may be redundant, given thius plugin uses yarn to build.